sigstore/_cli.py

```diff
@@ -692,3 +719,35 @@ def _get_identity_token(args: argparse.Namespace) -> Optional[str]:
             issuer,
         )
     return token
+
+
+class SplitCertificateChainError(Exception):
+    pass
+
+
+def _split_certificate_chain(chain_pem: str) -> List[bytes]:
```

Comment:
This is pretty nasty. Not sure if there is a cleaner way to do it.

I believe that the `load_pem...`functions in `cryptography` load the first valid PEM entry and leave the rest so I don't think they can help us here.

Reply:
Yeah, this isn't ideal, but it looks like neither `cryptography` nor `pyOpenSSL` exposes a "load an entire chain from PEMs" API. 

I _think_ the corresponding OpenSSL API is `SSL_CTX_get0_chain_certs`, so this might be worth a patch to pyOpenSSL in the medium term. But for now, something like this approach is fine 🙂

Reply:
NB: I opened https://github.com/pyca/cryptography/pull/7878 to add this API, so we'll be able to use it starting with cryptography 39 (assuming they accept it).

Reply:
Awesome, thanks @woodruffw.